docs/react-v9/contributing/patterns/basic-recomposition.md

```diff
@@ -0,0 +1,117 @@
+## Basic Recomposition
+
+Fluent UI React V9 leans into recomposition as one of its primary methods of customization. That is, reassemble parts of components to inherit most behaviors but also change some. We will go through an example where we implemented this with the `NavDivider` component below:
+
+1.  Change the default appearance prop to 'strong'
+    - The default Divider appearance happens to be the same color as the NavDrawer
+2.  Change some default styles
+    - FlexGrow needs to be changed from 1 to 0
+    - Add 4 pixels of top and bottom margin
+
+Assuming you've [scaffolded out the component](https://github.com/microsoft/fluentui/blob/8a3aa5f6200012d58ed80a833d8690d77935a48b/docs/react-v9/contributing/command-cheat-sheet.md?plain=1#L8), the best place to start is in the type definition.
+We want to inherit as much as we can from Divider, it simply becomes:
+
+```
+// NavDividerTypes.ts
+import type { DividerProps, DividerState } from '@fluentui/react-divider';
+
+// Remove the NavDivider slot type definition
+
+/**
+ * NavDivider Props
+ */
+export type NavDividerProps = DividerProps;
+
+/**
+ * State used in rendering NavDivider
+ */
+export type NavDividerState = DividerState;
+```
+
+We can also delete the `renderNavDivider.ts` file all together and update our wrapper component to call the base `renderDivider` function.
```

Comment:
Great doc! I think some context to let users know that the render func can be overridden if wanted, but it wasn't required here - some examples might be if we need to include our own context wrappers for the extended component and adding additional UI elements/slots or logic.

Reply:
Yep - I forsee an "Advanced Recomposition" document coming soon :-)